UnifiedSDK/UnifiedSDK/ViewController.swift

```diff
@@ -9,8 +9,7 @@
 import UIKit
 
 final class ViewController: UIViewController {
-
-    private let zendeskWrapper = ZendeskMessaging(themeColor: nil)
+    var zendesk: ZendeskMessaging?
```

Comment:
This is a bad practice. What if an integrator would like to show Messaging from 2 different view controllers in his app. Would it mean, he now has to declare & init the wrapper in 2 diff view controllers?